rebar.config

```diff
@@ -4,5 +4,5 @@
 {deps,
  [
   {lager, ".*", {git, "git://github.com/basho/lager.git", {tag, "2.0.3"}}},
-  {jsx, "1.4.5"}
+  {jsx,   ".*", {git, "git://github.com/talentdeficit/jsx.git", {tag, "2.9.0"}}}
```

Comment:
I would prefer this to remain a hex dependency, in fact both should be (to publish to hex) but the 1st one is on me

Reply:
@Licenser Got it. let me ask you that should I also patch Makefile and include rebar3 in libleofs in order to resolve a hex dependency like the following?

```diff
diff --git a/Makefile b/Makefile
index 0217c27..60787cf 100644
--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-REBAR = $(shell pwd)/rebar
+REBAR = $(shell pwd)/rebar3

 .PHONY: deps rel package

diff --git a/rebar b/rebar
deleted file mode 100755
index b0781c0..0000000
Binary files a/rebar and /dev/null differ
diff --git a/rebar.config b/rebar.config
index e67b89f..ca0f7d3 100644
--- a/rebar.config
+++ b/rebar.config
@@ -4,5 +4,5 @@
 {deps,
  [
   {lager, ".*", {git, "git://github.com/basho/lager.git", {tag, "2.0.3"}}},
-  {jsx,   ".*", {git, "git://github.com/talentdeficit/jsx.git", {tag, "2.9.0"}}}
+  {jsx, "2.9.0"}
  ]}.
```

Reply:
looks good tome!